nixos/modules/services/monitoring/rustdesk-server.nix

```diff
@@ -0,0 +1,96 @@
+{ lib, pkgs, config, ... }:
+{
+  options.services.rustdesk-server = with lib; with types; {
+    enable = mkEnableOption "enable RustDesk";
+
+    package = mkOption {
+      type = package;
+      description = "The directus package to use";
+      default = pkgs.rustdesk-server;
+    };
+
+    openFirewall = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Open the connection ports.
+        TCP (21115, 21116, 21117, 21118, 21119)
+        UDP (21116)
+      '';
+    };
+
+    relayIP = mkOption {
+      type = str;
+      description = ''
+        The RustDesk relay has to know its public facing IP
```

Comment:
I think this description should be reworded to describe what the option is, rather than why it exists and is required.

Reply:
You're right.